src/main/java/org/osc/manager/ism/api/IsmDeviceApi.java

```diff
@@ -373,5 +375,40 @@ private void updateMember(DeviceMemberEntity deviceMember, String vserverIpAddre
         deviceMember.setApplianceIp(contactIpAddress);
         deviceMember.setApplianceGateway(gateway);
         deviceMember.setApplianceSubnetMask(prefixLength);
+        if (deviceMember.getVersion() == null) {
+            SimpleDateFormat format = new SimpleDateFormat("yyyy/MM/dd-hh:mm");
+            String buildTime = format.format(new Date());
+            String ver = String.format("1.2(Build %d, %s)", 1234, buildTime);
+            deviceMember.setDiscovered(Boolean.TRUE);
+            deviceMember.setInspectionReady(Boolean.TRUE);
+            deviceMember.setVersion(ver);
+            deviceMember.setBrokerIp(null);
```

Comment:
@sudhirappaji I merged and looks like this is working. When you have a chance though on your next PR/commit can you make these three null values not null?  The UI is showing null for these three, ideally it would show something else. Thx.